XamlControlsGallery/ControlPages/CustomTitlebarPage.xaml

```diff
@@ -0,0 +1,212 @@
+<!--
+    //*********************************************************
+    //
+    // Copyright (c) Microsoft. All rights reserved.
+    // THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+    // ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+    // IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+    // PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+    //
+    //*********************************************************
+-->
+
+<Page
+    x:Class="AppUIBasics.ControlPages.CustomTitlebarPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:local="using:AppUIBasics"
+    xmlns:data="using:AppUIBasics.Data"
+    xmlns:wuxdata="using:Microsoft.UI.Xaml.Data"
+    mc:Ignorable="d">
+
+    <Page.Resources>
+        <x:Double x:Key="SwatchSize">32</x:Double>
+    </Page.Resources>
+    
+    <StackPanel>
+        <local:ControlExample HeaderText="User defined UIElement as custom titlebar for the window"
+                              CSharpSource="Window\CustomTitlebar\CustomTitlebarSample1.txt">
+            <local:ControlExample.Example>
+                <StackPanel Orientation="Vertical" Spacing="10">
+                    <TextBlock TextWrapping="WrapWholeWords">
+                        User can set a top-level UIElement (defined as appTitlebar here) as titlebar for the window. The system titlebar disappears and the chosen uielement starts acting like the titlebar. <LineBreak></LineBreak>
+                        The Background and Foreground Color dropdowns set the foreground and background of titlebar and caption buttons respectively.
+                    </TextBlock>
+                        <StackPanel Orientation="Horizontal" HorizontalAlignment="Stretch" VerticalAlignment="Top" Spacing="10">
+                            <Button x:Name="customTitlebar" Click="customTitlebar_Click"></Button>
+                        <TextBlock> Background Color </TextBlock>
+                        <SplitButton x:Name="myBgColorButton" AutomationProperties.Name="Foreground color" Padding="0" MinHeight="0" MinWidth="0" VerticalAlignment="Top">

```

Comment:
Automation Name for background and foreground buttons appear to be switched.

Reply:
fixed